osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -62,6 +68,18 @@ public SliderPath Path
             set
             {
                 PathBindable.Value = value;
+
+                RemoveAllFromNested(d => d is SliderTick);
+
+                foreach (var e in
```

Comment:
why does this code exist twice

Reply:
Where is the other place?

Reply:
the generate call is performed twice with the exact same arguments. should be extracted into a method.

`SliderEventGenerator.Generate(StartTime, SpanDuration, Velocity, TickDistance, Path.Distance, this.SpanCount(), LegacyLastTickOffset)`